index.html

```diff
@@ -1,14 +1,15 @@
 <!DOCTYPE html xmlns="http://www.w3.org/1999/xhtml" xmlns:fb="http://ogp.me/ns/fb#">
     <head>
         <meta http-equiv="content-type" content="text/html; charset=UTF8" />
-        <meta name="description" content="Electricity Map | Real-time visualisation of the climate footprint of electricity generation" />
+        <meta name="description" content="Electricity Map | Clean vs. dirty electricity production in Europe" />
```

Comment:
I don't like clean vs. dirty.. What did you dislike about the original title?
I liked the fact it had "realtime visualisation" and "climate footprint" (for google searches mostly)